.travis.yml

```diff
@@ -1,12 +1,19 @@
 language: php
 dist: trusty
 php:
-  - 7
-  - 7.1
+  - "7.1"
+  - "7.2"
 
-matrix:
-    allow_failures:
-        - php: hhvm
+env:
+  global:
+    # SAUCE_USERNAME
+    - secure: PBt4sGGD83QuBM96fc7QvNfN1+6SXs6sPlq/P1lDcYuSrgHEvvb1EtC1ioCIknYVfknB/cdb0RRKdrUAZfU+iYZwHbOaht+e4wWGqULTBXvLGa+HZoe3uLowRvwnIa7snWsyk+BfaIYF8yYZ2RAJ5uz9/7zCUH+iwgVydAE2YNijmQbapdoXjfi6YcPjRNHMjo2NBhISGs8CqJH2OcsuO3TwWl9nZKCJCpqGxwzF6K81hjNPPR6yODxy/Tttspc45yUG12rjg0iB9UYbCchPKo5ViLDvCRC4WI5qVPG0ZM5LdtbuIfWk0kBJDPwKBPommmQYZRKtticzTnKVV7Q/Qt9ddmKUBbRvc+Qh5gHf+AG0NMRemOUOEmMmSJpM+hS0K03+b9kU5FAfEHm2cNkSzh0c6Q7t9Cd22cw/8AVG6Is0iEytbYeXLDdh7y6s3v6X/PMuu9i1BaxcjKW3NsAK3zzlv/anYN6DSlGtvvjH4vh1Ifs9EqffA9QLj6onH9QDAhBCVehHEwHNXzZ9kE6NO/wFOOGPCPQulOLeAzVJAoIVWU9lBd7NAd6Z085o/iOiEJT6DzbuGNLDSb/Gz0+NOT52foxahxJbtwWtMG5yxHI7qoiHR5KUIHDM9RBUAHWqKaqsRbjdHDiDMp7Ptp9QFjyMA1jzlg2zDJn0e83nIRk=
```

Comment:
It could be placed directly in the Travis-CI settings tab and avoid to place it on code so you could rerun this PR without to ask for the secrets

Reply:
I've removed the variables so they can be set-up in Travis!